frontend/src/components/ApplicationSummaryTable.js

```diff
@@ -10,37 +12,52 @@ import Paper from '@mui/material/Paper';
 function createData(name, answer) {
   return { name, answer };
 }
-const rows = [
-  createData('Application ID:', 1234567),
-  createData('Last name / surname:', 'Smith'),
-  createData('First name / given name:', 'John'),
-  createData('Middle name(s):', 'David'),
-  createData('Email address:', 'jdsmith@shaw.ca'),
-  createData('Date of birth:', '19700906'),
-  createData('Street address:', '910 Government St.'),
-  createData('City:', 'Victoria BC'),
-  createData('Postal Code:', 'V8W 3Y8'),
-  createData('Social Insurance Number (SIN):', '******123'),
-  createData("B.C. Driver's Licence number:", 'DL:2222222'),
-  createData('Tax Year:', '2020'),
-  createData(
-    'Consent to Disclosure and Storage of, and Access to, Personal Information:',
-    'BCEIDJSMITH 20220504 12:04:01 PST'
-  ),
-  createData(
-    'Consent to Disclosure of Information from Income Tax Records:',
-    'BCEIDJSMITH 20220504 12:04:01 PST'
-  )
-];
 
 const ApplicationFormDetails = (props) => {
-  console.log(props);
+  const { id } = props;
+  const axiosInstance = useAxios();
+  const enabled = true;
+  const queryFn = () =>
+    axiosInstance.current.get(`/api/application-form/${id}`);
+
+  const { data } = useQuery({
```

Comment:
This should check the isLoading/isError state as well incase there's no data yet from the server or an error.